cloud/amazon/elasticache.py

```diff
@@ -50,8 +50,9 @@
     default: cache.m1.small
   num_nodes:
     description:
-      - The initial number of cache nodes that the cache cluster will have
+      - The initial number of cache nodes that the cache cluster will have. In case of redis this is always 1
```

Comment:
While the statement about redis is accurate, it feels unnecessary here. Which values are allowed in which parameters is really part of the functionality of AWS and not relevant to this specific module. So I think this change should be left out.